serving/docker/deepspeed.Dockerfile

```diff
@@ -16,6 +16,7 @@ ARG python_version=3.9
 ARG torch_version=2.0.1
 ARG torch_vision_version=0.15.2
 ARG deepspeed_wheel="https://publish.djl.ai/deepspeed/deepspeed-nightly-py2.py3-none-any.whl"
+ARG vllm_wheel="https://publish.djl.ai/vllm/vllm-0.0.0-cp39-cp39-linux_x86_64.whl"
```

Comment:
I would check with @frankfliu , vllm introduced a lot of dependencies you don't want, that's why we didn't have it for V5. If you already understand this, please feel free to ignore

Reply:
Yes, I discussed with Frank about this yesterday. So this solution to install wheel will not install the dependencies.

Reply:
Total size increase is 0.5GB.